drivers/rf-transceiver/hmc630x/hmc630x.c

```diff
@@ -522,6 +614,54 @@ int hmc630x_enable(struct hmc630x_dev *dev, bool enable)
 	return ret;
 }
 
+/* Is the chip power on? */
+int hmc630x_get_enable(struct hmc630x_dev *dev, bool *enable)
+{
+	int ret = -EFAULT;
+	uint8_t row, mask;
+	bool ena1, ena2;
+
+	if (dev->type == HMC6300) {
+		/* row4 */
+		ret = hmc630x_read_row(dev, HMC630X_ROW(HMC6300_DRIVER_PWRDN), &row);
+		if (ret)
+			return ret;
+		ena1 = (bool)row;
+
+		ret = hmc630x_read(dev, HMC630X_EN_SYNTH_LDO, &row);
+		if (ret)
+			return ret;
+		ena2 = (bool)row;
+	} else {
+		/* row0 */
+		mask = no_os_field_prep(HMC630X_MASK(HMC6301_IFVGA_PWRDN), 1);
```

Comment:
nit: i'd initialized only once the mask and avoid using `|=`

Reply:
i'm not sure i understand

Reply:
Something like this:
```
mask = no_os_field_prep(HMC630X_MASK(HMC6301_IFVGA_PWRDN), 1) |
            no_os_field_prep(HMC630X_MASK(HMC6301_MIXER_PWRDN), 1) |
	    no_os_field_prep(HMC630X_MASK(HMC6301_BBAMP_PWRDN_I), 1) |
	    no_os_field_prep(HMC630X_MASK(HMC6301_TRIPLER_PWRDN), 1) |
	    no_os_field_prep(HMC630X_MASK(HMC6301_LNA_PWRDWN), 1);
```

Reply:
ah i see,
how mandatory is this ? :D

Reply:
it's just a matter of preference I think :D